tests/scripts/check-raspbian

```diff
@@ -61,8 +71,13 @@ su -c 'RELEASE=1 script/bootstrap' pi
 # Pin CMake version to 3.10.3 for issue https://github.com/openthread/ot-br-posix/issues/728.
 # For more background, see https://gitlab.kitware.com/cmake/cmake/-/issues/20568.
 apt-get purge -y cmake
-pip3 install scikit-build
-pip3 install cmake==3.10.3
+if [ "${PYTHON3_BREAK_SYSTEM_PACKAGES}" == "1" ]; then
+    pip3 install scikit-build --break-system-packages
+    pip3 install cmake==3.10.3 --break-system-packages
+else
+    pip3 install scikit-build
+    pip3 install cmake==3.10.3
+fi
```

Comment:
Can you get rid of `--break-system-packages` by using virtualenv?

Reply:
Nice suggestion! Thanks 👍